<launch>
  <param name="use_sim_time" value="true"/>

  <group ns="object_detector">
    <rosparam file="$(find depth_image_extractor)/config/object_detection_rock.yaml"/>
  </group>
  <node name="object_detector" pkg="depth_image_extractor" type="detection_node" output="screen">
    <remap from="/camera/color/image_raw" to="/camera01/color/image_raw"/>
  </node>
</launch>